### PR TITLE
Fix AWS UPI s3 bucket issue

### PR DESCRIPTION
**Pull request by yunjiang29**

### CODE REVIEW

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-aws-upi-arm-f14

**Comment by yunjiang29** [comment]
/pj-rehearse pull-ci-openshift-installer-release-4.14-e2e-aws-ovn-upi

**Comment by yunjiang29** [comment]
cc @jianlinliu

File: ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh [hunk 1]
```diff
@@ -135,7 +135,9 @@ CONTROL_PLANE_STACK_NAME=${CLUSTER_NAME}-control-plane
 COMPUTE_STACK_NAME_PREFIX=${CLUSTER_NAME}-compute
 
 # Create s3 bucket for bootstrap and proxy ignition configs
-aws s3 mb s3://"${INFRA_STACK_NAME}"
+S3_BUCKET_URI="s3://${INFRA_STACK_NAME}"
+aws s3 mb "${S3_BUCKET_URI}"
+echo ${S3_BUCKET_URI} > ${SHARED_DIR}/s3_bucket_uri
```
Comment (jianlinliu) [inline ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh:140]: Maybe we can directly delete the s3 bucket in this upi-install script on `trap` lines

Comment (jianlinliu) [reply ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh:140]: Of course, this way should also work. On your call.

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-aws-upi-arm-f14

**Comment by yunjiang29** [comment]
```
Deleting bootstrap s3 bucket s3://ci-op-1683bjhi-17cbd-infra
delete: s3://ci-op-1683bjhi-17cbd-infra/bootstrap.ign
remove_bucket: ci-op-1683bjhi-17cbd-infra
```

**Pull request merged**